.travis.yml

```diff
@@ -15,3 +15,6 @@ jobs:
     - stage: pylint
       before_script: pip install pylint
       script: pylint --disable=C,R osm2gtfs
+
+    - stage: accra non regression test
+      script: python osm2gtfs/tests/accra_unittests.py
```

Comment:
Hmm.. could you make this generic, so it is named and functions in a way that it runs *all* unit tests? also those from the future like #78?

Reply:
Sorry for answering so late. I'll try to make this generic asap :)

Reply:
I've seen people use tox, but this is probably the easiest for now: https://setuptools.readthedocs.io/en/latest/setuptools.html#test

I tried `python setup.py test --test-suite=osm2gtfs.tests` and it worked. So just adding the `test_suite` to `setup.py` should do the trick.